python/dllib/src/bigdl/dllib/net/net_load.py

```diff
@@ -44,7 +44,10 @@ def get_python_class(self):
 
         jpackage_name = ".".join(self.jfullname.split(".")[:-1])
         pclass_name = self._get_py_name(self.jfullname.split(".")[-1])
-        base_module = self._load_ppackage_by_jpackage(jpackage_name)
+        if self.jfullname == "com.intel.analytics.bigdl.dllib.keras.Sequential":
```

Comment:
why do you need to handle it differently?

Reply:
The logic will be the same after we move sequential.py to models.py, will update it soon.

Reply:
Then you need to say in the header that "This file is adapted from ...", just like https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/orca/example/ray_on_spark/parameter_server/model.py